msal/application.py

```diff
@@ -1807,10 +1834,23 @@ def acquire_token_interactive(
             New in version 1.15.
 
         :param int parent_window_handle:
-            OPTIONAL. If your app is a GUI app running on modern Windows system,
-            and your app opts in to use broker,
-            you are recommended to also provide its window handle,
-            so that the sign in UI window will properly pop up on top of your window.
+            OPTIONAL.
+
+            * If your app does not opt in to use broker,
+              you do not need to provide a ``parent_window_handle`` here.
+
+            * If your app opts in to use broker,
+              ``parent_window_handle`` is required.
+
+              - If your app is a GUI app running on modern Windows system,
+                you are required to also provide its window handle,
+                so that the sign-in window will pop up on top of your window.
+              - If your app is a console app runnong on Windows system,
+                you can use a placeholder
+                ``PublicClientApplication.CONSOLE_WINDOW_HANDLE``.
+              - If your app is running on Mac,
+                you can use a placeholder
+                ``PublicClientApplication.CONSOLE_WINDOW_HANDLE``.
```

Comment:
FYI, those lines are the changes (or the lack thereof) of the `parent_window_handle` requirement between Windows and Mac.

The Windows behavior was influenced by the MsalRuntime's desire to make window handle explicit (CC @msamwils), and now on Mac this parameter remains required. (CC @kaisong1990 )

Downstream library may choose to make this parameter optional, by using MSAL's predefined CONSOLE_WINDOW_HANDLE placeholder as a default value. (CC @xiangyan99 )

Reply:
Sorry, you mean it is required on Mac because it is required on Windows?

I am a little confused.

Reply:
> Sorry, you mean it is required on Mac because it is required on Windows?
> 
> I am a little confused.

Put it this way. Most Python scripts are console apps and they shall ideally be cross-platform. How do we want those app developers to write their source code?

1. 
   ```python
   app.acquire_token_interactive(
       ...,
       parent_window_handle=app.CONSOLE_WINDOW_HANDLE,  # Same placeholder across all platforms, even though it can be a no-op on some platforms
       ...)
   ```
2. 
   ```python
   if sys.platform == "win32":
       app.acquire_token_interactive(
           ...,
           parent_window_handle=app.CONSOLE_WINDOW_HANDLE,  # Required on Windows
       ...)
   else:
       app.acquire_token_interactive(
           ...,
           # No need to specify parent_window_handle at all
       ...)
   ```